### ISSUE TITLE
Selecting autocomplete option leaves the "--" in-front of the var declaration for postcss files

### ISSUE THREAD

**Issue by JonJamesDesign** [paradigm-sy/vscode-css-variables-autocomplete#5]
First of all, _love_ the extension.

Got a slight issue: if I use it in a `.css` file with `postcss` language mode, and type:
```css
color: --
```
I get (as expected) a nice list of all my css variables to choose from.

The problem: when I select an autocomplete option from within a postcss file, the code updates to:
```css
color: --var(--var-i-chose)
```

Expected behaviour would be the code updates to:
```css
color: var(--var-i-chose)
```

**N.B:**
- Expected behaviour works for `.css` files with `css` language mode set.
- Only other css/postcss extension installed: https://marketplace.visualstudio.com/items?itemName=csstools.postcss

**Comment by equinusocio** [comment]
yes same issue here, i can't use it because i waste more time by going backward and deleting the `--` than writing the whole property

**Comment by paradigm-sy** [comment]
@JonJamesDesign @equinusocio it is actually not a good idea to use prefix `var` for your variables. Why do you use it?

**Comment by equinusocio** [comment]
The name of the variable doesn't matter (i dont use `var` as prefix but i have the same issue). With the `postcss` language when i type `--` and pick a suggestion, in turns into `--var(--my-prop)` (it doesn't replace the initial `--`)

**Comment by oraclo** [comment]
I have this problem too, would be nice if there is a fix.
